### PR TITLE
[MINOR][DOCS] Fix multi-line JavaDoc comments style in `BlockManagerMaster`

### PR DESCRIPTION
**Pull request by wzhfy**
### What changes were proposed in this pull request?

Fix multi-line JavaDoc comments style in `BlockManagerMaster`.
Also improve indentation in the `before` block in `BlockManagerReplicationSuite` for readability.

### How was this patch tested?

Only change some styles, should not affect tests.

### CODE REVIEW

**Review by sarutak — APPROVED** [review]

File: core/src/test/scala/org/apache/spark/storage/BlockManagerReplicationSuite.scala [hunk 1]
```diff
@@ -100,11 +100,14 @@ trait BlockManagerReplicationBehavior extends SparkFunSuite
 
     sc = new SparkContext("local", "test", conf)
     val blockManagerInfo = new mutable.HashMap[BlockManagerId, BlockManagerInfo]()
-    master = new BlockManagerMaster(rpcEnv.setupEndpoint("blockmanager",
-      new BlockManagerMasterEndpoint(rpcEnv, true, conf,
-        new LiveListenerBus(conf), None, blockManagerInfo, mapOutputTracker, isDriver = true)),
-      rpcEnv.setupEndpoint("blockmanagerHeartbeat",
-      new BlockManagerMasterHeartbeatEndpoint(rpcEnv, true, blockManagerInfo)), conf, true)
+    master = new BlockManagerMaster(
```
Comment (dongjoon-hyun) [inline core/src/test/scala/org/apache/spark/storage/BlockManagerReplicationSuite.scala:103]: Please revert this change from this PR. You should not mix like this in `DOCS` PR.

**Review by dongjoon-hyun — CHANGES_REQUESTED** [review]

File: core/src/main/scala/org/apache/spark/storage/BlockManagerMaster.scala [hunk 2]
```diff
@@ -56,8 +57,9 @@ class BlockManagerMaster(
     driverEndpoint.askSync[Seq[ReplicateBlock]](GetReplicateInfoForRDDBlocks(blockManagerId))
   }
 
-  /** Request removal of a dead executor from the driver endpoint.
-   *  This is only called on the driver side. Non-blocking
+  /**
+   * Request removal of a dead executor from the driver endpoint.
+   * This is only called on the driver side. Non-blocking.
```
[the remark this answers is not in the document]
Comment (wzhfy) [reply core/src/main/scala/org/apache/spark/storage/BlockManagerMaster.scala:62]: It's a very minor change, I just can't find a suitable place to fix it. If you think it's not worthy, I can close it.

**Comment by wzhfy** [comment]
Close this as it's too minor.
